Makefile

```diff
@@ -1,74 +1,29 @@
-VENVDIR = ./build/venv
-BINDIR = $(VENVDIR)/bin
-PYTHON = $(BINDIR)/python
-PIP = $(BINDIR)/pip
+HERE = $(shell pwd)
+VENV = ./venv
```

Comment:
What about looking first for an already activated virtualenv? See: https://github.com/mozilla-services/cliquet/blob/master/Makefile#L5